astropy/logger.py

```diff
@@ -211,7 +211,7 @@ def _showwarning(self, *args, **kwargs):
                     mod_name = mod.__name__
                     break
         else:  # pragma: py2
-            for name, mod in sys.modules.items():
+            for name, mod in list(sys.modules.items()):
```

Comment:
If it doesn't already, I feel like `six` should have `keys()`, `values()`, and `items()` functions (just as it has `iterkeys()`, `itervalues()`, and `iteritems()` functions) that reproduce the Python 2 behavior for those `dict` methods.